website/docs/releases/development-builds.md

```diff
@@ -155,17 +154,6 @@ document.addEventListener("DOMContentLoaded", () => {
       <img style="margin:auto;margin-left:0.1em;" src="../images/dots.svg">
     </td>
   </tr>
-  <tr>
-    <td id="msys2-build-link">
-      <img style="margin:auto;margin-left:0.1em;" src="../images/dots.svg">
-    </td>
-    <td id="msys2-build-version">
-      <img style="margin:auto;margin-left:0.1em;" src="../images/dots.svg">
-    </td>
-    <td id="msys2-build-date">
-      <img style="margin:auto;margin-left:0.1em;" src="../images/dots.svg">
-    </td>
-  </tr>
   <tr>
```

Comment:
I haven't had my coffee yet, but don't we want to keep the "msys2-build-*" items and remove the "windows-build-*" ones?

Reply:
I've just renamed the `msys2-build` CSS ID to `windows-build` so it's consistent with the `macos-build` and `linux-build` IDs. But I guess `windows-msys2-build` would be better if we intend to keep MSVC around in CI.